pkg/dependency/parser/golang/binary/parse.go

```diff
@@ -42,11 +44,22 @@ func (p *Parser) Parse(r xio.ReadSeekerAt) ([]types.Library, []types.Dependency,
 		return nil, nil, convertError(err)
 	}
 
-	libs := make([]types.Library, 0, len(info.Deps)+1)
-	libs = append(libs, types.Library{
-		Name:    "stdlib",
-		Version: strings.TrimPrefix(info.GoVersion, "go"),
-	})
+	libs := make([]types.Library, 0, len(info.Deps)+2)
+	libs = append(libs, []types.Library{
+		{
+			// Add the Go version used to build this binary.
+			Name:    "stdlib",
+			Version: strings.TrimPrefix(info.GoVersion, "go"),
+		},
+		{
+			// Add main module
+			Name: info.Main.Path,
+			// Only binaries installed with `go install` contain semver version of the main module.
+			// Other binaries use the `(devel)` version.
+			// See https://github.com/aquasecurity/trivy/issues/1837#issuecomment-1832523477.
+			Version: lo.Ternary(info.Main.Version != "(devel)", info.Main.Version, ""), // Use empty string instead of `(devel)`
```

Comment:
To avoid user confusion, we might want to add debug log where we hide `(devel)` version.
@knqyf263 wdyt?

Reply:
Agree

Reply:
added in https://github.com/aquasecurity/trivy/pull/6530/commits/8edf7c5b98bfa45fb026098a0ce3fe37cf9e1cc7